docs/index.js

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import ReactDOM from 'react-dom';
 import { Link, BrowserRouter as Router, Route } from 'react-router-dom';
-import CMS from '../src/components';
-import RouteProvider from '../src/components/routeProvider/ReactRouterProvider';
+import CMS from 'packages/CMS/src/components';
```

Comment:
I'm thinking maybe we should still name it `canner` instead of `CMS`?
Kind of like `apollo-server` did here

https://github.com/apollographql/apollo-server/tree/master/packages

Reply:
Yeah, folder name should be `canner`

Reply:
By the way, do you think we should change our exported modules? 
**current**
```js
import {CMS} from 'canner';
```

**changed**
```js
import Canner from 'canner';
```
or 
```js
import {Canner} from 'canner';
```

REF: https://github.com/Canner/canner/blob/master/src/index.js

Reply:
I think should change to `import Canner from 'canner';`, how do you think @wwwy3y3 ?

Reply:
```js
import Canner from 'canner';
```
It's better to import like this.

IMO, the cms component is our most critical module with most of our core functionalities. Thus, naming it to `canner` and import from default would make sense and straightforward to developers.

Reply:
I've changed the folder name to `canner` and also export default as Canner so we can use `import Canner from 'canner'`; Thanks!